sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/BatchServiceClientTestBase.java

```diff
@@ -317,7 +317,7 @@ static boolean waitForTasksToComplete(BatchClient batchClient, String jobId, int
         long elapsedTime = 0L;
 
         while (elapsedTime < expiryTimeInSeconds * 1000) {
-            PagedIterable<BatchTask> taskIterator = batchClient.listTasks(jobId, null, null, null, null,   Arrays.asList("id", "state"), null);
+            PagedIterable<BatchTask> taskIterator = batchClient.listTasks(jobId, null, null, null,   Arrays.asList("id", "state"), null);
```

Comment:
Was this supposed to use one of the new method signatures with the options bag object?

Reply:
Nope, this change is actually related to removing the ocp-date header and has nothing to do with the options bag change.